tests/app/helpers/test_template_helpers.py

```diff
@@ -431,6 +437,44 @@ def test_cookie_settings_url_context(
     assert result == expected
 
 
+@pytest.mark.parametrize(
+    "survey_config, cookie_present, expected",
+    [
+        (SurveyConfig(), True, f"{ACCOUNT_SERVICE_BASE_URL.replace('https://', '')}"),
+        (
+            BusinessSurveyConfig(),
+            True,
+            f"{ACCOUNT_SERVICE_BASE_URL.replace('https://', '')}",
+        ),
+        (
+            NorthernIrelandBusinessSurveyConfig(),
+            True,
+            f"{ACCOUNT_SERVICE_BASE_URL.replace('https://', '')}",
+        ),
+        (
+            SocialSurveyConfig(),
+            True,
+            f"{ACCOUNT_SERVICE_BASE_URL_SOCIAL.replace('https://', '')}",
```

Comment:
We should update `ACCOUNT_SERVICE_BASE_URL_SOCIAL` to `https://start.surveys.ons.gov.uk` then update the tests accordingly, as none of these tests show a cookie domain when the launching system is a sibling domain (which is what RH will be).

Reply:
Updated, thanks!